docusaurus/docs/users/howto/self-hosting.md

```diff
@@ -72,11 +86,13 @@ git@github.com:BobaBoard/selfhost-example.git
 
 ### flake.nix
```

Comment:
It doesn't allow me to comment above, but 

## Writing the NixOS configuration

(Add a quick refresher of what this configuration is and a quick overview of what people are going to do in this section. Bullet points are good.)

(edit later: I would also stress this is not the whole configuration to run bobaboard, but only the part that will allow you to connect to the server)

### Clone our sample repository 

(I'd specify this is a git thing. Like "Use Git to clone our repository". You giving the address is a bit weird because if they know how to clone then they know how to get the address, but if they don't know how to clone the address is useless. I would add a link to the GitHub instructions for cloning and then say "for example you can use `git clone git@github.com:BobaBoard/selfhost-example.git`."

Reply:
totally fair about people who understand git theoretically knowing how to get the address, i'm just terminally anxious about leaving instructional pages to go retrieve information elsewhere and like knowing i got the right thing 😅